CHANGELOG.md

```diff
@@ -2,6 +2,114 @@ TODO: This document was manually maintained so might be incomplete. The
+
+    * [https://github.com/kubernetes/kubernetes/pull/54748](https://github.com/kubernetes/kubernetes/pull/54748)
+
+* [k8s.io/api] Renamed `core/v1.StorageMediumHugepages` to `StorageMediumHugePages`
+
+    * [https://github.com/kubernetes/kubernetes/pull/54748](https://github.com/kubernetes/kubernetes/pull/54748)
+
+* Informers got a NewFilteredSharedInformerFactory to e.g. filter by namespace
+
+    * [https://github.com/kubernetes/kubernetes/pull/54660](https://github.com/kubernetes/kubernetes/pull/54660)
+
+* [k8s.io/api] The dynamic admission webhook is split into two kinds, mutating and validating.
+
+    * [https://github.com/kubernetes/kubernetes/pull/55282](https://github.com/kubernetes/kubernetes/pull/55282)
+
+* If you upgrade your client-go libs and use the `AppsV1() or Apps()` interface, please note that the default garbage collection behavior is changed.
+
+    * [https://github.com/kubernetes/kubernetes/pull/55148](https://github.com/kubernetes/kubernetes/pull/55148)
+
+**New Features:**
+
+* Certificate manager was moved from kubelet to client-go
+
+   * [https://github.com/kubernetes/kubernetes/pull/49654](https://github.com/kubernetes/kubernetes/pull/49654)
+
+* [k8s.io/apimachinery] `unstructued.Unstructured` got getters and setters
+
+    * [https://github.com/kubernetes/kubernetes/pull/51940](https://github.com/kubernetes/kubernetes/pull/51940)
+
+* [k8s.io/api] Workloads api types is promoted into new apps/v1 version
+
+    * [https://github.com/kubernetes/kubernetes/pull/53679](https://github.com/kubernetes/kubernetes/pull/53679)
+
+* [k8s.io/apimachinery] Introduces a polymorphic scale client, which supports scaling of resources in arbitrary API groups
+
+    * [https://github.com/kubernetes/kubernetes/pull/53743](https://github.com/kubernetes/kubernetes/pull/53743)
+
+* [k8s.io/apimachinery] `meta.MetadataAccessor` got API chunking support
+
+    * [https://github.com/kubernetes/kubernetes/pull/53768](https://github.com/kubernetes/kubernetes/pull/53768)
+
+* [k8s.io/api] Added `storage.k8s.io/v1alpha1` API group
+
+    * [https://github.com/kubernetes/kubernetes/pull/54463](https://github.com/kubernetes/kubernetes/pull/54463)
+
+* [k8s.io/api] `core/v1.Taint.TimeAdded` became a pointer
+
+   * [https://github.com/kubernetes/kubernetes/pull/43016](https://github.com/kubernetes/kubernetes/pull/43016)
+
+* [k8s.io/api] Added support for conditions in StatefulSet status
+
+    * [https://github.com/kubernetes/kubernetes/pull/55268](https://github.com/kubernetes/kubernetes/pull/55268)
+
+* [k8s.io/api] Added support for conditions in DaemonSet status
+
+    * [https://github.com/kubernetes/kubernetes/pull/55272](https://github.com/kubernetes/kubernetes/pull/55272)
+
+* [k8s.io/code-generator] `+groupName`tag added to allow unique Go identifiers in clientsets and informers
+
+    * [https://github.com/kubernetes/kubernetes/pull/54950](https://github.com/kubernetes/kubernetes/pull/54950)
+
+**Bug fixes and Improvements:**
```

Comment:
I think we can drop all changes but the one about `DefaultHardPodAffinitySymmetricWeight`. IMO we should only list really important bug fixes. What do you think?

Reply:
> IMO we should only list really important bug fixes. What do you think?

:+1: 

With `DefaultHardPodAffinitySymmetricWeight` moved to breaking change, we would have no bug fixes and improvements.